legacy_packages/cli/src/cli/index.ts

```diff
@@ -253,12 +253,9 @@ const main = async () => {
     .option("--forge", "Initialize as a Forge project.")
     .option("--hardhat", "Initialize as a Hardhat project.")
     .option("--extension", "Create a smart contract extension.")
-    .option("--react", "Initialize as a Create React App project.")
     .option("--next", "Initialize as a Next.js project.")
     .option("--vite", "Initialize as a Vite project.")
-    .option("--express", "Initialize as a Express project.")
     .option("--node", "Initialize as a Node project.")
```

Comment:
can prob remove node as well tbh

Reply:
Removed it from the menu so its not visible but can still be accessed with `--node` flag in case someone needs it